db/schema.rb

```diff
@@ -72,6 +72,15 @@
     t.index ["expires_date"], name: "index_customer_requests_on_expires_date", using: :btree
   end
 
+  create_table "customer_reviews", force: :cascade do |t|
+    t.integer  "customer_id"
+    t.integer  "company_id"
+    t.integer  "stars"
+    t.text     "body"
```

Comment:
nice, we'll want to add a limit to this field on the db, we don't want someone to troll the db with some crazy huge text bomb